docs/governance/templates/release.md

```diff
@@ -18,3 +18,6 @@ Images available with this release:
 - [gcr.io/agones-images/cpp-simple-server:{example-version}](https://gcr.io/agones-images/cpp-simple-server:{example-version})
 - [gcr.io/agones-images/udp-server:{example-version}](https://gcr.io/agones-images/udp-server:{example-version})
 - [gcr.io/agones-images/xonotic-example:{example-version}](https://gcr.io/agones-images/xonotic-example:{example-version})
+- [`helm install agones/agones --version {example-version}][https://agones.dev/chart/stable/agones-{example-version}.tgz]
```

Comment:
Small issue - link should be in `()` not `[]` :smile: 

Should this be in it's own section above, rather in images?